target_chains/ethereum/sdk/js/src/examples/EvmRelay.ts

```diff
@@ -55,6 +55,7 @@ async function run() {
     providerOrUrl: network,
   });
 
+  // @ts-ignore
```

Comment:
this is a weird one: the package versions changed, and in the process somehow the `HDWalletProvider` is no longer type-compatible with the argument to `Web3`. I tried changing the versions a bit and couldn't get this to work out. I validated that the script still runs properly though.